prism-highlighter.html

```diff
@@ -59,7 +59,12 @@
         return;
       }
 
+      if(event.detail._highlighted === true) {
```

Comment:
I used `stopPropagation` originally but changed it to this in case, for some strange reason, someone calls `fire(...)` twice with the same event detail object.

I guess its too much of an edge case though, so can replace this with `stopPropagation`.